index.js

```diff
@@ -32,15 +32,15 @@ function loader (version) {
       }
     }
 
-    static equal (item1, item2) {
+    static equal (item1, item2, matchStackSize = true) {
       if (item1 == null && item2 == null) {
         return true
       } else if (item1 == null) {
         return false
       } else if (item2 == null) {
         return false
       } else {
-        return item1.type === item2.type &&
+        return (matchStackSize ? item1.type === item2.type : true) &&
```

Comment:
This is not doing the right thing.
I guess you mean the below line

Reply:
fixed